src/asciidoc/docs/develop/editor.adoc

```diff
@@ -1,123 +1,156 @@
 
-* *Sticker* - Displays how your template will look if it's imbedded in another template.
 
-* *Thumbnail* - Displays how your template will look if it's shared on social media.
+=== Previews outside the Code Editor
 
-=== Remote previews
+You can also preview a Koji outside the Code Editor.
 
-The preview pane provides these options for previewing a template outside of the Koji editor.
+* The QR code button image:{imagesDir}/QRcode.svg[QR code button] displays a QR code for your Koji.
+Scan this code to preview the Koji on your mobile device.
 
-* The QR code button image:QRcode.png[QR code button] displays a QR code for the template.
-You can scan this code to preview the template on your mobile device.
+* The remote staging button image:{imagesDir}/remote-staging.svg[Remote staging button] opens a preview of your Koji in a new browser tab.
 
-* The remote staging button image:remote-staging.png[Remote staging button] opens a preview of your template in a new browser tab.
 
-=== Launching the debugger
+== Debugging
 
-*Launch Debugger* opens the debugger in a new browser tab.
+To open the debugger in a new browser tab, click *Launch Debugger*.
 
 
-[.faq]
 == Tips
-[id="faq-nodejsnpmver"]
-.What versions of Node.js and npm do Kojis run on?
+
+* While editing a plain text file, such as .html or .json, you can open menus with additional editing options.
++
+.Learn how...
 [%collapsible]
 ====
-include::{includespath}/misc.adoc[tag=nodejsnpmver]
+[.singlespaced]
+. In the Code Editor, move the focus to the editing pane where the contents of the text file is displayed.
+. Right-click anywhere in the editing pane for the contextual menu.
+. Press *F1* for the Command Palette menu.
 ====
 
-[id="faq-revert-kojiversion"]
-.How do I revert to an old version of my template?
+* You can revert to an old version of your project.
++
+.Learn how...
 [%collapsible]
 ====
-To revert to a previous commit,
-
-include::{includespath}/steps-codeeditor.adoc[tag=openproject-template2edit]
-
-include::{includespath}/steps-codeeditor.adoc[tag=expand-terminal]
-
+[.singlespaced]
+. Open your project in the Code Editor.
+. Expand the terminal pane.
 . Run `git log` and copy the hash code of the commit you want to restore.
-
-. In the terminal panel, enter `git reset --hard <commit-hash>`.
+. In the terminal pane, enter `git reset --hard <commit-hash>`.
 +
 WARNING: You will lose all the changes you made after the commit you restore.
-
-TIP: Publish more frequently for a more detailed commit history.
 ====
 
-[id="faq-reopen-frontend-terminal"]
-.How do I reopen the frontend terminal?
+* Publish more frequently for a more detailed commit history.
+
+* If you accidentally closed the frontend terminal, you can reopen it.
++
+.Learn how...
 [%collapsible]
 ====
-To re-open the frontend terminal:
-
-. Click on the [.tabname]`New` tab.
-
+[.singlespaced]
+. In the Code Editor, click *+ New tab*.
 . Run:
 +
 ....
```

Comment:
This formatting looks a bit odd to me because it's code font without a background - something we haven't done elsewhere. We probably need a standard for how we handle "console" text. We've been using code (for span) or code block (for multi-line blocks).

Reply:
Replaced "...." with "----".

Reply:
Hmm, that did something strange to the formatting. It just put a border around it rather than adding a background. Let's add a source tag to use the standard code block. E.g.

```
+
[source, console]
----
cd frontend
npm start
----
```